src/service/transactionManagement.ts

```diff
@@ -12,17 +12,18 @@ export const createTransaction = (amount:number, sender:user, receiver:user, sta
 
 export const getBalance = (BlockChain:blockChain, User:user) : number => (
   BlockChain.transactions
-    .filter((Transaction) => (Transaction.sender === User || Transaction.receiver === User))
-    .reduce(((accumulator, Transaction) => ((Transaction.receiver === User)
-      ? (accumulator + Transaction.amount)
-      : (accumulator - Transaction.amount))), 0));
+    .filter((Transaction) => (
+      Transaction.sender?.PrivateKey === User.PrivateKey
+      || Transaction.receiver?.PrivateKey === User.PrivateKey))
+    .reduce(((accumulator, Transaction) => (
```

Comment:
a variable name shouldn't start with an uppercase.
fix the linter to avoid theses errors